logger/logger.go

```diff
@@ -0,0 +1,81 @@
+package logger
```

Comment:
I'd like to wrap all the logic inside a Logger struct.
`type Logger struct {
...
}

func (logger *Logger) Debug(...) {}
...`

And then inject the logger from the main to all the components.

Reply:
Could you provide a usage example, who exactly the struct should be injected to the all components?

Reply:
There are two ways : 

- Inject the log in the context -> https://thegodev.com/store-logger-in-context/
- Inject the logger in the struct constructor 

From my point of view the best way is to inject it into the context -> https://www.kaznacheev.me/posts/en/where-to-place-logger-in-golang/

Reply:
thank you for the link. I am going to go through it

Reply:
Are you fixing it in this PR ? Can I close the conversation ?

Reply:
I would like to fix it in another PR. Because the discussion becomes a bit overloaded